app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,45 @@
+class BookingsController < ApplicationController
+
+  def index
+    @bookings = Booking.all
```

Comment:
For now, we only need the new, create, and params method. 
The users shouldn´t be able to delete or edit the booking, so it would be the best, if we just leave it simple for now.
If we see, that we need more methods, we could implement them later on

Reply:
Genau.
Thanks Paul.